### ISSUE TITLE
Test_Udf fails at go test

### ISSUE THREAD

**Issue by rogers0** [viant/toolbox#29]
Hi there,

I'm working with @creekorful to package this project into Debian.
Debian build system usually run go test after building the package, so I find Test_Udf fails, whatever has network access.
Here's the log:

```
=== RUN   Test_Udf
    Test_Udf: map_test.go:288:
                Error Trace:    map_test.go:288
                Error:          Not equal:
                                expected: "2008"
                                actual  : "2009"

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -2008
                                +2009
                Test:           Test_Udf
    Test_Udf: map_test.go:296:
                Error Trace:    map_test.go:296
                Error:          Not equal:
                                expected: "2008"
                                actual  : "2009"

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -2008
                                +2009
                Test:           Test_Udf
    Test_Udf: map_test.go:343:
                Error Trace:    map_test.go:343
                Error:          Not equal:
                                expected: "2008"
                                actual  : "2009"

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -2008
                                +2009
                Test:           Test_Udf
--- FAIL: Test_Udf (0.00s)
```

**Comment by adranwit** [comment]
Updated test plan

**Comment by adranwit** [comment]
patched issue

**Comment by rogers0** [comment]
I confirmed this is fixed on version 0.32.2
Thanks for your support!
